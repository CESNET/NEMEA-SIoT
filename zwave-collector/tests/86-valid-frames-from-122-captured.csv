time TIMESTAMP,uint8 CHANNEL,bytes FRAME
2019-07-29T17:38:45.935,3,cdc3498a0341011101700609020000a01a00000000000000000000000000000000000000000000000000000000000000000000
2019-07-29T17:38:46.058,3,cdc3498a03410211017006090200011abc000000
2019-07-29T17:38:46.109,3,cdc3498a03410211017006090200011abc000000
2019-07-29T17:38:46.180,2,cdc3498a03510210017006090200001c00
2019-07-29T17:38:46.929,3,cdc3498a0341030e012001ff083a00000000
2019-07-29T17:38:47.234,3,cdc3498a0341030e012001ff083a000000
2019-07-29T17:38:47.623,3,cdc3498a0341040e012001ff80f6800000
2019-07-29T17:38:47.827,2,cdc3498a0351040d012001ffb600
2019-07-29T17:38:47.865,2,cdc3498a03450215012000fa40000000008007fc98
2019-07-29T17:38:50.485,3,cdc3498a03410515017105000001fe0e100002fd940000
2019-07-29T17:38:50.630,3,cdc3498a03410615017105000001fe0e100001c300000000
2019-07-29T17:38:50.669,3,cdc3498a03410615017105000001fe0e100001c3000000
2019-07-29T17:38:50.765,2,cdc3498a03510614017105000001fe0e100003dc00
2019-07-29T17:38:52.042,3,cdc3498a03410711013105012200fc1666000000
2019-07-29T17:38:52.042,2,cdc3498a03510710013105012200fc9d00
2019-07-29T17:38:52.070,3,cdc3498a03410711013105012200fc1666000008
2019-07-29T17:38:52.095,3,cdc3498a03410811013105012200fc8e32000000
2019-07-29T17:38:52.144,2,cdc3498a03510810013105012200fc9200
2019-07-29T17:38:52.146,3,cdc3498a03410811013105012200fc8e32000010
2019-07-29T17:38:54.971,3,cdc3498a0341091001310505014c702f40000000
2019-07-29T17:38:55.018,2,cdc3498a0351090f01310505014c1b00
2019-07-29T17:38:55.035,3,cdc3498a03410a1001310505014cb85a000000
2019-07-29T17:38:55.154,2,cdc3498a03510a0f01310505014c180002
2019-07-29T17:38:55.187,2,cdc3498a03450517012000fa4000000000c4141405320401
2019-07-29T17:38:56.977,3,cdc3498a03410b0e01800364ec5400000000
2019-07-29T17:38:57.074,3,cdc3498a03410c0e018003642415400010
2019-07-29T17:38:57.101,3,cdc3498a03410c0e01800364241540000000
2019-07-29T17:38:57.143,2,cdc3498a03510c0d018003648700
2019-07-29T17:38:59.363,3,cdc3498a03410d11013105030a001dd6fe000000
2019-07-29T17:38:59.410,3,cdc3498a03410d11013105030a001dd6fe00000020
2019-07-29T17:38:59.495,3,cdc3498a03410e11013105030a001dfbba00000040
2019-07-29T17:38:59.570,2,cdc3498a03510e10013105030a001d5f00
2019-07-29T17:38:59.602,2,cdc3498a03450718012000fa4000000000c4140c28007740
2019-07-29T17:39:01.251,3,cdc3498a03410f100131051b010021ce00000000
2019-07-29T17:39:01.283,2,cdc3498a03510f0f0131051b01004f00
2019-07-29T17:39:01.324,3,cdc3498a034101100131051b0100bfdf400000
2019-07-29T17:39:01.397,2,cdc3498a03450817012000fa4000000000c4146c04037802
2019-07-29T17:39:08.902,3,cdc3498a02410114017105000001fe0c2c00e08e80000001
2019-07-29T17:39:08.907,2,cdc3498a02510113017105000001fe0c2c01d200
2019-07-29T17:39:08.925,3,cdc3498a02410214017105000001fe0c2c00ebb0000008
2019-07-29T17:39:08.999,3,cdc3498a02410214017105000001fe0c2c00ebb00180
2019-07-29T17:39:09.045,2,cdc3498a02510213017105000001fe0c2c01d400
2019-07-29T17:39:09.285,2,cdc3498a0245011b012000fa3103000000e20a000003fc1858007408
2019-07-29T17:39:11.060,1,cdc34985fcfefddf00fefe2c63fe9080af43392cc2b9b8c1bfe7e747c2d2bbd353a0
2019-07-29T17:39:12.496,1,cdc3498a03010320ff0101d39c0121015e86725985737184803031707a5aef4c4c
2019-07-29T17:39:13.379,3,cdc3498a0241030e01800364a6c540000040
2019-07-29T17:39:13.434,2,cdc3498a0251030d018003648900
2019-07-29T17:39:13.495,3,cdc3498a0241040c
2019-07-29T17:39:13.501,3,cdc3498a0241040e018003646e84000004
2019-07-29T17:39:13.510,2,cdc3498a0251040d018003648e00
2019-07-29T17:39:13.543,2,cdc3498a02450215012000fa4000000002000d9078
2019-07-29T17:39:13.629,2,cdc3498a02450215012000fa31030000010006c8d801
2019-07-29T17:39:18.638,3,cdc3498a02410115017105000001fe0e0002077d8e800004
2019-07-29T17:39:18.660,2,cdc3498a02510114017105000001fe0e00040fec00
2019-07-29T17:39:18.670,3,cdc3498a02410215017105000001fe0e000206e2c400000040
2019-07-29T17:39:18.785,2,cdc3498a02510214017105000001fe0e00040fe000
2019-07-29T17:39:18.823,2,cdc3498a0245011c012000fa4000000001c414000007f838001037d004
2019-07-29T17:39:22.914,2,cdc3498a02510313017105000001fe0e0601fe00
2019-07-29T17:39:22.930,3,cdc3498a02410314017105000001fe0e06017a36800004
2019-07-29T17:39:23.001,2,cdc3498a02510413017105000001fe0e0601f000
2019-07-29T17:39:23.043,2,cdc3498a0245021b012000fa4000000001c414000007f8381803c004
2019-07-29T17:39:23.043,3,cdc3498a02410414017105000001fe0e06016aa0000001
2019-07-29T17:39:23.106,3,cdc3498a02410414017105000001fe0e06016aa000000008
2019-07-29T17:39:27.033,3,cdc3498a02410514017105000001fe0e06016c4a800001
2019-07-29T17:39:27.106,3,cdc3498a02410514017105000001fe0e06016c4a000001
2019-07-29T17:39:27.139,2,cdc3498a02510513017105000001fe0e0601f200
2019-07-29T17:39:27.151,3,cdc3498a02410614017105000001fe0e06016774000004
2019-07-29T17:39:27.281,2,cdc3498a02510613017105000001fe0e0601f400
2019-07-29T17:39:27.314,2,cdc3498a0245031b012000fa4000000001c414000007f8381803c8
2019-07-29T17:39:27.642,2,fe90821d0353090d012503ff8200
2019-07-29T17:39:27.669,2,cdc3498a0245031b012000fa3103000000e20a000003fc1c0c002c00
2019-07-29T17:39:31.360,3,cdc3498a02410714017105000001fe0e0601619e80000020
2019-07-29T17:39:31.430,3,cdc3498a02410814017105000001fe0e06014658000020
2019-07-29T17:39:31.502,3,cdc3498a02410814017105000001fe0e06014658000020
2019-07-29T17:39:31.549,2,cdc3498a02510813017105000001fe0e0601e800
2019-07-29T17:39:31.579,2,cdc3498a
2019-07-29T17:39:35.928,3,cdc3498a0241090e0180036420c7400000
2019-07-29T17:39:36.030,2,cdc3498a02510a0d018003648000
2019-07-29T17:39:41.317,1,cdc3498a0201011cff0101539c0107015e855986725a738071847a4b4b
2019-07-29T17:39:43.578,2,cdc3498a02050128ff2000fa31030000000246a738020e02bd0ab30ce4b4e700e308f400000002c4
2019-07-29T17:39:54.292,2,cdc3498a02050328ff2000fa4000000000048d4e70041c057a156619c969ce01c611e80000000608
2019-07-29T17:40:02.348,2,cdc3498a02050428ff2000fa4000000000048d4e70041c057a156619c969ce01c611e8000000063004
2019-07-29T17:40:07.001,3,cdc3498a0241020e01800364e365400003
2019-07-29T17:40:07.021,3,cdc3498a0241030e01800364a6c5400008
2019-07-29T17:40:07.115,2,cdc3498a0251030d018003648900
2019-07-29T17:40:07.147,2,cdc3498a02450515012000fa4000000002000d906402
2019-07-29T17:40:27.531,3,fe90821d0341010e01250300152800000000
2019-07-29T17:40:27.648,3,fe90821d03810111010010042503003b9140000010
2019-07-29T17:40:27.684,3,fe90821d03810111010010042503003b91400000
2019-07-29T17:40:27.751,2,fe90821d0381011001101004250300be00
2019-07-29T17:40:27.844,3,fe90821d0341020e01250300dbc8000000
2019-07-29T17:40:27.890,3,fe90821d0341020e01250300dbc800000000
2019-07-29T17:40:27.959,2,fe90821d0351020d012503007400
2019-07-29T17:40:32.146,3,fe90821d0341030e012503009e6800000000
2019-07-29T17:40:32.196,3,fe90821d038102110100100425030016d5000000
2019-07-29T17:40:32.230,2,fe90821d0381031001101004250300bc00
2019-07-29T17:40:32.246,3,fe90821d038102110100100425030016d5000000
2019-07-29T17:40:32.282,2,fe90821d0381031001101004250300bc00
2019-07-29T17:40:32.312,2,fe90821d0381031001101004250300bc00
2019-07-29T17:40:32.358,3,fe90821d0341040e012503005629400000
2019-07-29T17:40:32.431,3,fe90821d0341040e01250300562900000000
2019-07-29T17:40:32.473,2,fe90821d0351040d012503007200
2019-07-29T17:40:36.620,1,fe90821d03010522ff0101d39c0110015e22855970565a7a72328e717398312586ebeb
2019-07-29T17:40:36.667,3,fe90821d0341060e06
2019-07-29T17:40:36.714,2,fe90821d0351060d012503ff8f00
2019-07-29T17:40:36.719,3,fe90821d0341060e012503ffc399000000
2019-07-29T17:40:36.795,3,fe90821d03810311010010042503ffe30600000000
2019-07-29T17:40:36.846,2,fe90821d03810510011010042503ff4500
2019-07-29T17:40:36.877,2,fe90821d03810510011010042503ff450004
2019-07-29T17:40:36.989,3,fe90821d0341070e012503ff0c72000000
2019-07-29T17:40:37.046,2,fe90821d03450615012000fa4000000000940ffc6400
2019-07-29T17:40:41.193,3,fe90821d0341080e012503ff433a00000000
2019-07-29T17:40:41.300,2,fe90821d0351080d012503ff8100
2019-07-29T17:40:41.334,3,fe90821d03810411010010042503ff52ad000000
2019-07-29T17:40:41.395,3,fe90821d03810411010010042503ff52ad40000020
2019-07-29T17:40:41.427,2,fe
2019-07-29T17:40:41.439,2,fe90821d03810710011010042503ff470000
2019-07-29T17:40:41.536,3,fe90821d0341090e012503ff069a000000
2019-07-29T17:40:41.641,2,fe90821d0351090d012503ff8000
2019-07-29T17:40:41.642,2,fe90821d0353090d012503ff8200
2019-07-29T17:40:41.673,2,fe90821d03450815012000fa4000000000940ffc5c00
