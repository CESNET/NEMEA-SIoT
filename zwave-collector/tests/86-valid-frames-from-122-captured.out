3,2019-07-29T17:38:45.935,3452127626,1,3,0,1,0,0,1,17,0,"Configuration","",,700609020000,"","Singlecast"
3,2019-07-29T17:38:46.058,3452127626,1,3,0,1,0,0,2,17,0,"Configuration","",,700609020001,"","Singlecast"
3,2019-07-29T17:38:46.109,3452127626,1,3,0,1,0,0,2,17,0,"Configuration","",,700609020001,"","Singlecast"
3,2019-07-29T17:38:46.929,3452127626,1,3,0,1,0,0,3,14,0,"Basic","",,2001ff,"","Singlecast"
3,2019-07-29T17:38:47.234,3452127626,1,3,0,1,0,0,3,14,0,"Basic","",,2001ff,"","Singlecast"
3,2019-07-29T17:38:47.623,3452127626,1,3,0,1,0,0,4,14,0,"Basic","",,2001ff,"","Singlecast"
3,2019-07-29T17:38:47.827,3452127626,1,2,0,1,0,0,4,13,0,"Basic","",,2001ff,"","Singlecast"
3,2019-07-29T17:38:50.485,3452127626,1,3,0,1,0,0,5,21,0,"Alarm","",,7105000001fe0e100002,"","Singlecast"
3,2019-07-29T17:38:52.042,3452127626,1,3,0,1,0,0,7,17,0,"Sensor Multilevel","",,3105012200fc,"","Singlecast"
3,2019-07-29T17:38:52.042,3452127626,1,2,0,1,0,0,7,16,0,"Sensor Multilevel","",,3105012200fc,"","Singlecast"
3,2019-07-29T17:38:52.070,3452127626,1,3,0,1,0,0,7,17,0,"Sensor Multilevel","",,3105012200fc,"","Singlecast"
3,2019-07-29T17:38:52.095,3452127626,1,3,0,1,0,0,8,17,0,"Sensor Multilevel","",,3105012200fc,"","Singlecast"
3,2019-07-29T17:38:52.144,3452127626,1,2,0,1,0,0,8,16,0,"Sensor Multilevel","",,3105012200fc,"","Singlecast"
3,2019-07-29T17:38:52.146,3452127626,1,3,0,1,0,0,8,17,0,"Sensor Multilevel","",,3105012200fc,"","Singlecast"
3,2019-07-29T17:38:54.971,3452127626,1,3,0,1,0,0,9,16,0,"Sensor Multilevel","",,310505014c,"","Singlecast"
3,2019-07-29T17:38:55.018,3452127626,1,2,0,1,0,0,9,15,0,"Sensor Multilevel","",,310505014c,"","Singlecast"
3,2019-07-29T17:38:55.035,3452127626,1,3,0,1,0,0,10,16,0,"Sensor Multilevel","",,310505014c,"","Singlecast"
3,2019-07-29T17:38:55.154,3452127626,1,2,0,1,0,0,10,15,0,"Sensor Multilevel","",,310505014c,"","Singlecast"
3,2019-07-29T17:38:56.977,3452127626,1,3,0,1,0,0,11,14,0,"Battery","",,800364,"","Singlecast"
3,2019-07-29T17:38:57.074,3452127626,1,3,0,1,0,0,12,14,0,"Battery","",,800364,"","Singlecast"
3,2019-07-29T17:38:57.101,3452127626,1,3,0,1,0,0,12,14,0,"Battery","",,800364,"","Singlecast"
3,2019-07-29T17:38:57.143,3452127626,1,2,0,1,0,0,12,13,0,"Battery","",,800364,"","Singlecast"
3,2019-07-29T17:38:59.363,3452127626,1,3,0,1,0,0,13,17,0,"Sensor Multilevel","",,3105030a001d,"","Singlecast"
3,2019-07-29T17:38:59.410,3452127626,1,3,0,1,0,0,13,17,0,"Sensor Multilevel","",,3105030a001d,"","Singlecast"
3,2019-07-29T17:38:59.495,3452127626,1,3,0,1,0,0,14,17,0,"Sensor Multilevel","",,3105030a001d,"","Singlecast"
3,2019-07-29T17:38:59.570,3452127626,1,2,0,1,0,0,14,16,0,"Sensor Multilevel","",,3105030a001d,"","Singlecast"
3,2019-07-29T17:39:01.251,3452127626,1,3,0,1,0,0,15,16,0,"Sensor Multilevel","",,31051b0100,"","Singlecast"
3,2019-07-29T17:39:01.283,3452127626,1,2,0,1,0,0,15,15,0,"Sensor Multilevel","",,31051b0100,"","Singlecast"
3,2019-07-29T17:39:01.324,3452127626,1,3,0,1,0,0,1,16,0,"Sensor Multilevel","",,31051b0100,"","Singlecast"
2,2019-07-29T17:39:08.902,3452127626,1,3,0,1,0,0,1,20,0,"Alarm","",,7105000001fe0c2c00,"","Singlecast"
2,2019-07-29T17:39:08.925,3452127626,1,3,0,1,0,0,2,20,0,"Alarm","",,7105000001fe0c2c00,"","Singlecast"
2,2019-07-29T17:39:08.999,3452127626,1,3,0,1,0,0,2,20,0,"Alarm","",,7105000001fe0c2c00,"","Singlecast"
3,2019-07-29T17:39:12.496,3452127626,0,1,0,255,0,0,3,32,0,"System","Node Info",,0101d39c0121015e86725985737184803031707a5aef,"","Broadcast"
2,2019-07-29T17:39:13.379,3452127626,1,3,0,1,0,0,3,14,0,"Battery","",,800364,"","Singlecast"
2,2019-07-29T17:39:13.434,3452127626,1,2,0,1,0,0,3,13,0,"Battery","",,800364,"","Singlecast"
2,2019-07-29T17:39:13.501,3452127626,1,3,0,1,0,0,4,14,0,"Battery","",,800364,"","Singlecast"
2,2019-07-29T17:39:13.510,3452127626,1,2,0,1,0,0,4,13,0,"Battery","",,800364,"","Singlecast"
2,2019-07-29T17:39:18.638,3452127626,1,3,0,1,0,0,1,21,0,"Alarm","",,7105000001fe0e000207,"","Singlecast"
2,2019-07-29T17:39:18.670,3452127626,1,3,0,1,0,0,2,21,0,"Alarm","",,7105000001fe0e000206,"","Singlecast"
2,2019-07-29T17:39:22.930,3452127626,1,3,0,1,0,0,3,20,0,"Alarm","",,7105000001fe0e0601,"","Singlecast"
2,2019-07-29T17:39:23.043,3452127626,1,3,0,1,0,0,4,20,0,"Alarm","",,7105000001fe0e0601,"","Singlecast"
2,2019-07-29T17:39:23.106,3452127626,1,3,0,1,0,0,4,20,0,"Alarm","",,7105000001fe0e0601,"","Singlecast"
2,2019-07-29T17:39:27.033,3452127626,1,3,0,1,0,0,5,20,0,"Alarm","",,7105000001fe0e0601,"","Singlecast"
2,2019-07-29T17:39:27.106,3452127626,1,3,0,1,0,0,5,20,0,"Alarm","",,7105000001fe0e0601,"","Singlecast"
2,2019-07-29T17:39:27.151,3452127626,1,3,0,1,0,0,6,20,0,"Alarm","",,7105000001fe0e0601,"","Singlecast"
3,2019-07-29T17:39:27.642,4270883357,1,2,0,1,0,0,9,13,0,"","",,,"","Ack"
2,2019-07-29T17:39:31.360,3452127626,1,3,0,1,0,0,7,20,0,"Alarm","",,7105000001fe0e0601,"","Singlecast"
2,2019-07-29T17:39:31.430,3452127626,1,3,0,1,0,0,8,20,0,"Alarm","",,7105000001fe0e0601,"","Singlecast"
2,2019-07-29T17:39:31.502,3452127626,1,3,0,1,0,0,8,20,0,"Alarm","",,7105000001fe0e0601,"","Singlecast"
2,2019-07-29T17:39:35.928,3452127626,1,3,0,1,0,0,9,14,0,"Battery","",,800364,"","Singlecast"
2,2019-07-29T17:39:36.030,3452127626,1,2,0,1,0,0,10,13,0,"Battery","",,800364,"","Singlecast"
2,2019-07-29T17:39:41.317,3452127626,0,1,0,255,0,0,1,28,0,"System","Node Info",,0101539c0107015e855986725a738071847a,"","Broadcast"
2,2019-07-29T17:40:07.001,3452127626,1,3,0,1,0,0,2,14,0,"Battery","",,800364,"","Singlecast"
2,2019-07-29T17:40:07.021,3452127626,1,3,0,1,0,0,3,14,0,"Battery","",,800364,"","Singlecast"
2,2019-07-29T17:40:07.115,3452127626,1,2,0,1,0,0,3,13,0,"Battery","",,800364,"","Singlecast"
3,2019-07-29T17:40:27.531,4270883357,1,3,0,1,0,0,1,14,0,"Switch Binary","",,250300,"","Singlecast"
3,2019-07-29T17:40:27.648,4270883357,0,3,4,1,0,1,1,17,3,"Switch Binary","",04,250300,"Request","Singlecast"
3,2019-07-29T17:40:27.684,4270883357,0,3,4,1,0,1,1,17,3,"Switch Binary","",04,250300,"Request","Singlecast"
3,2019-07-29T17:40:27.751,4270883357,0,2,4,1,4,1,1,16,3,"Switch Binary","",04,250300,"Request","Singlecast"
3,2019-07-29T17:40:27.844,4270883357,1,3,0,1,0,0,2,14,0,"Switch Binary","",,250300,"","Singlecast"
3,2019-07-29T17:40:27.890,4270883357,1,3,0,1,0,0,2,14,0,"Switch Binary","",,250300,"","Singlecast"
3,2019-07-29T17:40:27.959,4270883357,1,2,0,1,0,0,2,13,0,"Switch Binary","",,250300,"","Singlecast"
3,2019-07-29T17:40:32.146,4270883357,1,3,0,1,0,0,3,14,0,"Switch Binary","",,250300,"","Singlecast"
3,2019-07-29T17:40:32.196,4270883357,0,3,4,1,0,1,2,17,3,"Switch Binary","",04,250300,"Request","Singlecast"
3,2019-07-29T17:40:32.230,4270883357,0,2,4,1,4,1,3,16,3,"Switch Binary","",04,250300,"Request","Singlecast"
3,2019-07-29T17:40:32.246,4270883357,0,3,4,1,0,1,2,17,3,"Switch Binary","",04,250300,"Request","Singlecast"
3,2019-07-29T17:40:32.282,4270883357,0,2,4,1,4,1,3,16,3,"Switch Binary","",04,250300,"Request","Singlecast"
3,2019-07-29T17:40:32.312,4270883357,0,2,4,1,4,1,3,16,3,"Switch Binary","",04,250300,"Request","Singlecast"
3,2019-07-29T17:40:32.358,4270883357,1,3,0,1,0,0,4,14,0,"Switch Binary","",,250300,"","Singlecast"
3,2019-07-29T17:40:32.431,4270883357,1,3,0,1,0,0,4,14,0,"Switch Binary","",,250300,"","Singlecast"
3,2019-07-29T17:40:32.473,4270883357,1,2,0,1,0,0,4,13,0,"Switch Binary","",,250300,"","Singlecast"
3,2019-07-29T17:40:36.620,4270883357,0,1,0,255,0,0,5,34,0,"System","Node Info",,0101d39c0110015e22855970565a7a72328e717398312586,"","Broadcast"
3,2019-07-29T17:40:36.714,4270883357,1,2,0,1,0,0,6,13,0,"Switch Binary","",,2503ff,"","Singlecast"
3,2019-07-29T17:40:36.719,4270883357,1,3,0,1,0,0,6,14,0,"Switch Binary","",,2503ff,"","Singlecast"
3,2019-07-29T17:40:36.795,4270883357,0,3,4,1,0,1,3,17,3,"Switch Binary","",04,2503ff,"Request","Singlecast"
3,2019-07-29T17:40:36.846,4270883357,0,2,4,1,4,1,5,16,3,"Switch Binary","",04,2503ff,"Request","Singlecast"
3,2019-07-29T17:40:36.877,4270883357,0,2,4,1,4,1,5,16,3,"Switch Binary","",04,2503ff,"Request","Singlecast"
3,2019-07-29T17:40:36.989,4270883357,1,3,0,1,0,0,7,14,0,"Switch Binary","",,2503ff,"","Singlecast"
3,2019-07-29T17:40:41.193,4270883357,1,3,0,1,0,0,8,14,0,"Switch Binary","",,2503ff,"","Singlecast"
3,2019-07-29T17:40:41.300,4270883357,1,2,0,1,0,0,8,13,0,"Switch Binary","",,2503ff,"","Singlecast"
3,2019-07-29T17:40:41.334,4270883357,0,3,4,1,0,1,4,17,3,"Switch Binary","",04,2503ff,"Request","Singlecast"
3,2019-07-29T17:40:41.395,4270883357,0,3,4,1,0,1,4,17,3,"Switch Binary","",04,2503ff,"Request","Singlecast"
3,2019-07-29T17:40:41.439,4270883357,0,2,4,1,4,1,7,16,3,"Switch Binary","",04,2503ff,"Request","Singlecast"
3,2019-07-29T17:40:41.536,4270883357,1,3,0,1,0,0,9,14,0,"Switch Binary","",,2503ff,"","Singlecast"
3,2019-07-29T17:40:41.641,4270883357,1,2,0,1,0,0,9,13,0,"Switch Binary","",,2503ff,"","Singlecast"
3,2019-07-29T17:40:41.642,4270883357,1,2,0,1,0,0,9,13,0,"","",,,"","Ack"
